backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -303,14 +326,22 @@ public static String convertEthnicityToDatabaseValue(String ethnicity) {
   public static String convertRaceToDatabaseValue(String race) {
     Map<String, String> displayValueToDatabaseValue =
         Map.ofEntries(
-            Map.entry("american indian or alaska native", "native"),
+            Map.entry(NATIVE_LITERAL, NATIVE_DB_VALUE),
+            Map.entry(NATIVE_CODE, NATIVE_DB_VALUE),
             Map.entry(ASIAN_LITERAL, ASIAN_LITERAL),
-            Map.entry("black or african american", "black"),
-            Map.entry("native hawaiian or other pacific islander", "pacific"),
+            Map.entry(ASIAN_CODE, ASIAN_LITERAL),
+            Map.entry(BLACK_LITERAL, BLACK_DB_VALUE),
+            Map.entry(BLACK_CODE, BLACK_DB_VALUE),
+            Map.entry(PACIFIC_ISLANDER_LITERAL, PACIFIC_DB_VALUE),
+            Map.entry(PACIFIC_ISLANDER_CODE, PACIFIC_DB_VALUE),
             Map.entry(WHITE_LITERAL, WHITE_LITERAL),
+            Map.entry(WHITE_CODE, WHITE_LITERAL),
             Map.entry(OTHER_LITERAL, OTHER_LITERAL),
-            Map.entry("ask but unknown", REFUSED_LITERAL),
-            Map.entry(UNKNOWN_LITERAL, UNKNOWN_LITERAL));
+            Map.entry(OTHER_RACE_CODE, OTHER_LITERAL),
+            Map.entry(ASK_LITERAL, REFUSED_LITERAL),
+            Map.entry(ASK_CODE, REFUSED_LITERAL),
+            Map.entry(UNKNOWN_LITERAL, UNKNOWN_LITERAL),
+            Map.entry(UNKNOWN_CODE, UNKNOWN_LITERAL));
```

Comment:
When I upload a [patient](https://dev2.simplereport.gov/app/patient/3c51c6d6-083d-4526-9080-bac45becef40?facility=1a0139b9-fc23-450b-9b6c-cd081e5cea9d) with "UNK" or "unknown", their race fields are missing when I look at it again. 
![image](https://user-images.githubusercontent.com/10108172/227981174-5760e2ca-5e67-4be1-87aa-0c2045c5545a.png)